test/extended/prometheus/collection_profiles.go

```diff
@@ -120,7 +121,7 @@ var _ = g.Describe("[sig-instrumentation][OCPFeatureGate:MetricsCollectionProfil
 			}
 
 			return nil
-		}).Should(o.BeNil())
+		}, pollTimeout, pollInterval).Should(o.BeNil())
```

Comment:
Owing to the changes introduced in https://github.com/openshift/origin/pull/29428. 

Please `/lgtm` again, thanks.